### DISCUSSION TITLE
【部桃遭駭危人命】全院個資外洩、竄改點滴險害命　部桃遭對岸駭客攻陷

### DISCUSSION THREAD

**Discussion by 1104030360** [1101-GS4719-C/Python-Community-GS4719-C#1291] [科技新知]
沒想到駭客入侵也能直接影響病患的生命安全，真的是太可怕了..... 這篇文章推薦給家中有在醫院工作的人或是有生病待在醫院的人，我認為我們不只在銀行、政府機構方面的資安系統要注意，在醫院的資安系統也要多下功夫
https://www.mirrormedia.mg/story/20221206soc001/

**Comment by YoYooHuang** [comment]
這也讓我們知道，不能過於仰賴科技，也讓我們反省，科技所帶來的不只是便利，其中一定參雜一些負面作用。期望未來的科技風氣能夠朝正向持續發展，為人類帶來更多正面影響！
